github_pr.py

```diff
@@ -215,31 +215,30 @@ def main():
 
     args = vars(parser.parse_args())
 
-    if args['action'] == 'create':
+    if 'action' in args and args['action'] == 'create':
         github_create_pr(**args)
 
-    elif args['action'] == 'list':
+    elif 'action' in args and args['action'] == 'list':
         github_list_prs(**args)
 
-    elif args['action'] == 'merge':
-        if args['number']:
+    elif 'action' in args and args['action'] == 'merge':
+        if 'number' in args and args['number']:
             github_merge_pr_by_number(**args)
         else:
             github_merge_pr_by_branch(**args)
 
-    elif args['action'] == 'comment':
+    elif 'action' in args and args['action'] == 'comment':
         github_comment_pr(**args)
 
-    elif args['action'] == 'delete':
+    elif 'action' in args and args['action'] == 'delete':
         github_delete_pr(**args)
 
-    elif args['action'] == 'update':
+    elif 'action' in args and args['action'] == 'update':
         github_update_pr(**args)
 
     gh = Github(args['token'])
-    if not args['numberonly'] and not args['noratelimit']:
+    if (('numberonly' in args) and not args['numberonly']) or (('noratelimit' in args) and not args['noratelimit']):
```

Comment:
okay, so if either of them is false then you want to print the message. I just wanted to make it clear that this was a change in behavior from the previous logic.